pkg/agent/cniserver/pod_configuration.go

```diff
@@ -542,10 +542,8 @@ func (pc *podConfigurator) disconnectInterceptedInterface(podName, podNamespace,
 		return nil
 	}
 	for _, ip := range containerConfig.IPs {
-		if err := pc.routeClient.UnMigrateRoutesFromGw(&net.IPNet{
-			IP:   ip,
-			Mask: net.CIDRMask(32, 32),
-		}, ""); err != nil {
+		ipNet := util.Get32BitsMaskIPNet(ip)
```

Comment:
The util doesn't sound generic, maybe you can use `netlink.NewIPNet` or copy the function to antrea's util too.

Reply:
Good suggestion. Updated.